### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by fokuspokus**
- added a description for Google OAuth authorization
- added links to binaries to the main readme

### CODE REVIEW

**Review by ValeraFinebits — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -47,14 +47,31 @@ First of all, clone the project repository to your machine:
...
+
+#### Linux
+[Eppie.CLI-linux-arm.zip](https://github.com/Eppie-io/Eppie-CLI/releases/latest/download/Eppie.CLI-linux-arm.zip)  
+[Eppie.CLI-linux-arm64.zip](https://github.com/Eppie-io/Eppie-CLI/releases/latest/download/Eppie.CLI-linux-arm64.zip)  
+[Eppie.CLI-linux-x64.zip](https://github.com/Eppie-io/Eppie-CLI/releases/latest/download/Eppie.CLI-linux-x64.zip)
+
+#### MacOS
+[Eppie.CLI-osx-arm64.zip](https://github.com/Eppie-io/Eppie-CLI/releases/latest/download/Eppie.CLI-osx-arm64.zip)  
+[Eppie.CLI-osx-x64.zip](https://github.com/Eppie-io/Eppie-CLI/releases/latest/download/Eppie.CLI-osx-x64.zip)
+```
```
Comment (al-kau) [inline README.md:73]: line: 73 
remove ```

Comment (al-kau) [reply README.md]: done

File: docs/images/gmail/gmail3.png
Comment (al-kau) [inline docs/images/gmail/gmail3.png:1]: Maybe need to hide _'Project number'_ and _'Project ID'_ value in **Project Info** section

Comment (al-kau) [reply docs/images/gmail/gmail3.png:1]: done [7e88a42](https://github.com/Eppie-io/Eppie-CLI/pull/167/commits/7e88a42f9e200aac0108a9cf1bb500a64edb58dd)

**Comment by al-kau** [comment]
Related to the issue #158

**Pull request merged**